src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -192,21 +192,25 @@ private void fireInventoryUpdated() {
 	private List<Lot> lotsSaved = new ArrayList<>();
 	private HashMap<Integer, Lot> lotsDeleted = new HashMap<>();
 	List<MedicalInventoryRow> inventoryRowsToDelete = new ArrayList<>();
-	private String[] pColums = { MessageBundle.getMessage("angal.inventory.id.col").toUpperCase(),
+	private String[] columsNames = { MessageBundle.getMessage("angal.inventory.id.col").toUpperCase(),
 			MessageBundle.getMessage("angal.common.code.txt").toUpperCase(),
 			MessageBundle.getMessage("angal.inventory.product.col").toUpperCase(),
 			MessageBundle.getMessage("angal.inventory.newlot.col").toUpperCase(),
 			MessageBundle.getMessage("angal.wardpharmacy.lotnumber.col").toUpperCase(),
 			MessageBundle.getMessage("angal.medicalstock.duedate.col").toUpperCase(),
-			MessageBundle.getMessage("angal.inventoryrow.theoreticqty.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.theoreticqty.col").toUpperCase(),
 			MessageBundle.getMessage("angal.inventoryrow.realqty.col").toUpperCase(),
-			MessageBundle.getMessage("angal.inventoryrow.unitprice.col").toUpperCase(),
-			MessageBundle.getMessage("angal.inventory.totalprice").toUpperCase() };
-	private int[] pColumwidth = { 50, 50, 200, 100, 100, 100, 100, 80, 80, 80 };
+			MessageBundle.getMessage("angal.inventory.unitcost.col").toUpperCase(),
+			MessageBundle.getMessage("angal.inventory.totalcost.col").toUpperCase() };
+	private int[] columwidth = { 50, 50, 200, 100, 100, 100, 100, 80, 80, 80 };
 	private boolean[] columnEditable = { false, false, false, false, false, false, false, true, false, false };
 	private boolean[] columnEditableView = { false, false, false, false, false, false, false, false, false, false };
-	private boolean[] pColumnVisible = { false, true, true, true, !GeneralData.AUTOMATICLOT_IN, true, true, true, GeneralData.LOTWITHCOST,
+	private boolean[] columnVisible = { false, true, true, true, !GeneralData.AUTOMATICLOT_IN, true, true, true, GeneralData.LOTWITHCOST,
 			GeneralData.LOTWITHCOST };
+	private boolean[] columnCentered = { false, false, false, true, true, true, true, true, true, true };
+	private boolean[] columnDecimalNumber = { false, false, false, false, false, false, false, false, true, true };
+	private Class< ? >[] columnsClasses = { String.class, Integer.class, String.class, String.class, String.class, LocalDate.class, Integer.class,
+			Integer.class, BigDecimal.class, BigDecimal.class };
 	private MedicalInventory inventory = null;
```

Comment:
```suggestion
	private MedicalInventory inventory;
```
Initialization to `null` is redundant